docs/HowTo/Deploy/Lite-Network-Monitor.md

```diff
@@ -3,164 +3,179 @@ description: Alethio EthStats Lite network monitor
-* Node logs. Node logs display the data sent by a node
-* Block history.  Block history provides the ability to go back in time and playback the block propagation
- through the nodes
- 
-## Components 
-
-EthStats Lite consists of: 
+  - Information about the last received block such as block number, block hash, transaction count,
+    uncle count, block time and propagation time.
+  - Connected peers, whether the node is mining, hash rate, latency, and uptime.
+* Charts for Block Time, Block Difficulty, Block Gas Limit, Block Uncles, Block Transactions,
+  Block Gas Used, Block Propagation Histogram, and Top Miners.
+* IP-based geolocation overview.
+* Node logs, which display the data sent by a node.
+* Block history, which provides the ability to go back in time and playback the block propagation
+  through the nodes.
 
-* [Server](https://github.com/Alethio/ethstats-network-server). Consumes node data received from the 
-client. 
+## Components
 
-* [Client](https://github.com/Alethio/ethstats-cli). A client must be started for each node in the network.
-The client extracts data from the node and sends it to the server
+EthStats Lite consists of a:
 
-* [Dashboard](https://github.com/Alethio/ethstats-network-dashboard). Dashboard displaying [statistics](#statistics).
+* [Server](https://github.com/Alethio/ethstats-network-server). The server consumes node data
+  received from the client.
+* [Client](https://github.com/Alethio/ethstats-cli). Each node in the network requires you to start
+  a client. The client extracts data from the node and sends it to the server.
+* [Dashboard](https://github.com/Alethio/ethstats-network-dashboard). The dashboard displays
+  [statistics](#statistics).
 
-## Pre-requisities 
+## Pre-requisities
 
 [Docker](https://docs.docker.com/install/)
 
 !!! tip
+
     EthStats Lite has a number of dependencies. Using Docker is the easiest way to
     use EthStats Lite with Besu.
-    
+
     The [EthStats CLI](https://github.com/Alethio/ethstats-cli),
-    [EthStats Network Server](https://github.com/Alethio/ethstats-network-server), and [EthStats Network
-    Dashboard](https://github.com/Alethio/ethstats-network-dashboard) documentation describes how to 
-    install EthStats Lite tools. 
+    [EthStats Network Server](https://github.com/Alethio/ethstats-network-server), and
+    [EthStats Network Dashboard](https://github.com/Alethio/ethstats-network-dashboard)
+    documentation describes how to install the EthStats Lite tools.
 
-## Running EthStats Lite for a Besu Node
+## Running EthStats Lite for a Besu node
 
 !!! important
-    This example describes how to run EthStats Lite for a single Besu node.
-    To run EthStats Lite for a network of nodes, a [client](#3-client) must be started for each node. 
+
+    This example describes how to run EthStats Lite for a single Besu node. To run EthStats Lite
+    for a network of nodes, each node requires you to start a [client](#3-client) for each node.
```

Comment:
Having _each node_ twice sounds a little circular - could it be worded differently?

Reply:
Changed to:
"Running EthStats Lite for a network requires starting a [client](#3-client) for each node in the network."

Reply:
Something like "Run one dedicated EthStat client for each node on the network to be able to connect to EthStats Lite server"?

Reply:
"For a network, run one dedicated [EthStats Lite client](#3-client) for each node on the network."

Reply:
Perhaps even without he additionnal "network" at the end: "For a network, run one dedicated EthStats Lite client for each Besu node." Maybe specifying what node we talk about for the sentence to be absolutely clear?

Reply:
Changed, as you suggested @NicolasMassart